logger/awslogs/logger.go

```diff
@@ -102,13 +102,13 @@ func (la *LoggerArgs) RunLogDriver(ctx context.Context, config *logging.Config,
 	}
 
 	if la.globalArgs.Mode == logger.NonBlockingMode {
-		debug.SendEventsToJournal(logger.DaemonName, "Starting log streaming for non-blocking mode awslogs driver",
-			journal.PriInfo, 0)
+		debug.SendEventsToLog(logger.DaemonName, "Starting log streaming for non-blocking mode awslogs driver",
+			"info", 0)
```

Comment:
Can you please save these strings to constants like `info`, `error`, etc. instead of having plain strings there?